crates/typst-render/src/text.rs

```diff
@@ -18,20 +18,19 @@ pub fn render_text(canvas: &mut sk::Pixmap, state: State, text: &TextItem) {
         let id = GlyphId(glyph.id);
         let offset = x + glyph.x_offset.at(text.size).to_f32();
 
-        if is_color_glyph(&text.font, glyph) {
+        if is_outlinable(&text.font, glyph) {
```

Comment:
Has this been tested on a mac w/ Apple's emoji font?  The copy of Apple Color Emoji I have has mono outlines and color bitmaps.  I suspect this will prefer mono emoji.

Reply:
I tested it and it does work. If you look at the logic of `is_outlinable`, it prefers color tables. The naming doesn't make this clear enough, I'll rename it to `should_outline`.